skopt/dummy_opt.py

```diff
@@ -4,16 +4,19 @@
 import inspect
 import numbers
 import numpy as np
+from time import time
```

Comment:
`from collections import Callable` missing. Can you also make a common test to exercise the API (checking that an exception is raised if `callable` isn't a list of callables or a callable).